### PR TITLE
Fixes #918 move from stedolan/jq to jqlang/jq

### LINKED ISSUE #918: Repository Move from stedolan/jq to jqlang/jq.  301 received and not followed
Jq repository is moving from stedolan/jq to jqlang/jq and a redirection is implemented by stedolan

https://github.com/jqlang/jq/issues/2594

## Current Behaviour
ark get jq receive a 301 on jq releases listing and can't follow it.

## Expected Behaviour
Update ark jq repository link

# Possible Solution
alert and follow 301 with a new follow-redirect ark option

## Context
docker image

```
arkade version
```
0.9.18

### PR DESCRIPTION
**Pull request by Obsecurus**
## Description
Moves JQ repo to new owner `jqlang` as the 301 redirect fails. 

## Motivation and Context

- [ ] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?
`make test`
`make e2e`

```
➜  arkade git:(jq-redirect-fix) ./hack/test-tool.sh jq
+ ./arkade get jq --arch arm64 --os darwin --quiet
+ file /Users/matt/.arkade/bin/jq
/Users/matt/.arkade/bin/jq: Mach-O 64-bit executable x86_64
+ rm /Users/matt/.arkade/bin/jq
+ echo

+ ./arkade get jq --arch x86_64 --os darwin --quiet
+ file /Users/matt/.arkade/bin/jq
/Users/matt/.arkade/bin/jq: Mach-O 64-bit executable x86_64
+ rm /Users/matt/.arkade/bin/jq
+ echo

+ ./arkade get jq --arch x86_64 --os linux --quiet
+ file /Users/matt/.arkade/bin/jq
/Users/matt/.arkade/bin/jq: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, for GNU/Linux 2.6.32, BuildID[sha1]=c159e8a134ee222019dfbac0a1af4bc94b302eb0, with debug_info, not stripped
+ rm /Users/matt/.arkade/bin/jq
+ echo

+ ./arkade get jq --arch aarch64 --os linux --quiet
+ file /Users/matt/.arkade/bin/jq
/Users/matt/.arkade/bin/jq: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), statically linked, for GNU/Linux 3.2.0, BuildID[sha1]=eac34d86f35a499d8b42cb9c2532c0180239a1e1, with debug_info, not stripped
+ rm /Users/matt/.arkade/bin/jq
+ echo

+ ./arkade get jq --arch x86_64 --os mingw --quiet
+ file /Users/matt/.arkade/bin/jq.exe
/Users/matt/.arkade/bin/jq.exe: PE32+ executable (console) x86-64, for MS Windows
+ rm /Users/matt/.arkade/bin/jq.exe
+ echo
```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [x] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ about at the end of this document.
 To add the sign-off, simply run:
 
 ```bash
-git commit --global user.name "Full Name"
-git commit --global user.email "you@example.com"
+git config --global user.name "Full Name"
+git config --global user.email "you@example.com"
```
Comment (Obsecurus) [inline CONTRIBUTING.md:30]: This was just a documentation bug.

**Review by joebowbeer — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (alexellis) [reply CONTRIBUTING.md:30]: Oh thanks for catching that.

**Comment by Obsecurus** [comment]
> 

hmm let me peek at that

**Comment by Obsecurus** [comment]
> Why didn't the unit tests need to change?
> 
> Are there none? (if so, can we add the required ones?)
> 
> We then just need this running
> 
> https://github.com/alexellis/arkade/blob/master/hack/test-tool.sh

updated the description with that output

**Comment by alexellis** [comment]
This output looks like it may be incorrect:

```
+ ./arkade get jq --arch arm64 --os darwin --quiet
+ file /Users/matt/.arkade/bin/jq
/Users/matt/.arkade/bin/jq: Mach-O 64-bit executable x86_64
+ rm /Users/matt/.arkade/bin/jq
+ echo
```

Or is there no arm64 binary for Darwin for jq?

**Review by alexellis — APPROVED** [review]
Approved

**Comment by alexellis** [comment]
Maybe the jq app just lacks those architectures, or maybe the template is wrong?

**Pull request merged**